developer-guides/realtime-api/the-message-object/keyboard/README.md

```diff
@@ -0,0 +1,30 @@
+# Keyboard
```

Comment:
Would this be ignored on the desktop, or implemented differently?

Reply:
This would be mobile only.